test/Interpreter/multi_payload_shifting.swift

```diff
@@ -1,7 +1,6 @@
 // RUN: %empty-directory(%t)
 
-// RUN: %target-build-swift -o %t/a.out %s
-// RUN: %target-run %t/a.out 2>&1
+// RUN: %target-run-simple-swift %s
```

Comment:
Ah, sorry, this substitution has the `%s` in it already. You also don't need the `%empty-directory` in this case.